### ISSUE TITLE
[Bug]: MicrosoftFSLogixApps has stopped working

### ISSUE THREAD

**Issue by EnsorsBryan** [aaronparker/evergreen#259]
### What happened?

Seems that Microsoft may have changed a URL.

### Version

2110.467

### What PowerShell edition/s are you running Evergreen on?

Windows PowerShell

### Which operating system/s are you running Evergreen on?

Windows Server 2016+

### Have you reviewed the documentation?

- [X] Troubleshooting at: https://stealthpuppy.com/evergreen/troubleshoot/
- [X] Known issues at: https://stealthpuppy.com/evergreen/issues/

### Verbose output

```shell
Get-EvergreenApp MicrosoftFSLogixApps -Verbose
VERBOSE: Get-EvergreenApp: Function exists: C:\Program
Files\WindowsPowerShell\Modules\Evergreen\2110.467\Apps\Get-MicrosoftFSLogixApps.ps1.
VERBOSE: Get-EvergreenApp: Dot sourcing: C:\Program
Files\WindowsPowerShell\Modules\Evergreen\2110.467\Apps\Get-MicrosoftFSLogixApps.ps1.
VERBOSE: Get-FunctionResource: read application resource strings from [C:\Program
Files\WindowsPowerShell\Modules\Evergreen\2110.467\Manifests\MicrosoftFSLogixApps.json]
VERBOSE: Get-EvergreenApp: Calling: Get-MicrosoftFSLogixApps.
VERBOSE: Resolve-SystemNetWebRequest: Attempting to resolve: https://aka.ms/fslogix/downloadpreview.
WARNING: Resolve-SystemNetWebRequest: Error at URI: https://aka.ms/fslogix/downloadpreview.
WARNING: Resolve-SystemNetWebRequest: Response: Exception calling "GetResponse" with "0" argument(s): "The remote
server returned an error: (404) Not Found.".
WARNING: Resolve-SystemNetWebRequest: For troubleshooting steps see: https://stealthpuppy.com/evergreen/troubleshoot/.
VERBOSE: Resolve-SystemNetWebRequest: Attempting to resolve: https://aka.ms/fslogix_download.
WARNING: Resolve-SystemNetWebRequest: Error at URI: https://aka.ms/fslogix_download.
WARNING: Resolve-SystemNetWebRequest: Response: Exception calling "GetResponse" with "0" argument(s): "The remote
server returned an error: (404) Not Found.".
WARNING: Resolve-SystemNetWebRequest: For troubleshooting steps see: https://stealthpuppy.com/evergreen/troubleshoot/.
Get-EvergreenApp: Failed to capture output from: Get-MicrosoftFSLogixApps.
At C:\Program Files\WindowsPowerShell\Modules\Evergreen\2110.467\Public\Get-EvergreenApp.ps1:82 char:17
+ ...             Throw "$($MyInvocation.MyCommand): Failed to capture outp ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Get-EvergreenAp...oftFSLogixApps.:String) [], RuntimeException
    + FullyQualifiedErrorId : Get-EvergreenApp: Failed to capture output from: Get-MicrosoftFSLogixApps.
```

**Comment by ItzLevvie** [comment]
New download link can be found at https://docs.microsoft.com/en-us/fslogix/install-ht#download-fslogix

**Comment by aaronparker** [comment]
Fixed in 7dd91a1227d73453de6646538aa563042fcd3b22
